packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -11,6 +11,10 @@ import 'binary_messenger.dart';
 import 'message_codec.dart';
 import 'message_codecs.dart';
 
+/// Returns the number of invoke methods waiting for return value.
```

Comment:
Can you add a little more details to this doc comment? It should probably link to [invokeMethod] and explain a little bit why this value is useful.

Reply:
Done. PTAL.